content/applications/sales/crm/performance/win_loss.rst

```diff
@@ -1,26 +1,249 @@
-    :align: center
+Follow the steps below to create a basic win/loss report using the above criteria:
+
+1. Navigate to :menuselection:`CRM app --> Reporting --> Pipeline`. At the *Pipeline Analysis* page,
+   click on the :guilabel:`Search bar` to open a menu of filters and grouping options.
+
+.. image:: win_loss/filters-for-basic-win-loss-report.png
+   :align: center
+   :alt: The Search menu containing the filters for a basic win/loss report.
+
+2. In the *Search* menu, under the *Group By* heading, click :guilabel:`Stage`. Under the *Filters*
+   heading, click :guilabel:`Add Custom Filter`.
+3. In the resulting pop-up menu, click the current filter (by default, it is set to
+   :guilabel:`Country`) and select :guilabel:`Active` from the drop-down. The rule should now read
+   "`Active` `is` `set`".
+4. Click :guilabel:`New Rule`, and once more change the default filter to :guilabel:`Active`. Click
+   on the last part of this rule and change it to read "`Active` `is` `not set`."
+5. Click :guilabel:`Add`.
+
+.. image:: win_loss/add-custom-active-filter.png
+   :align: center
+   :alt: The Add Custom Filter menu showing two rules: (1) Active is set, and (2) Active is not set.
+
+Click off the *Search* menu to reveal the report, which now displays the total :guilabel:`Count` of
+leads, whether won or lost, grouped by their stage in the CRM pipeline. Hover over a section of the
+report to see the number of leads in that stage.
+
+.. image:: win_loss/basic-win-loss-report.png
+   :align: center
+   :alt: A basic win/loss report showing all leads whether won or lost grouped by stage.
+
+Advanced filters and groupings
+------------------------------
+
+The *Pipeline Analysis* can be customized with various filters and group by options. After
+creating a basic win/loss report, click on the :guilabel:`Search bar` and consider adding advanced
+filters and groupings to customize the report for different purposes. Some useful options
+include:
+
+- **Created On**: Adjusting this filter to a different period of time such as the last 30 days
+  or the last quarter can provide more timely results.
+- **Add Custom Filter**: Clicking :guilabel:`Add Custom Filter` and scrolling through the numerous
+  options in the drop-down opens up additional search criteria like :guilabel:`Last Stage Update` or
+  :guilabel:`Lost Reason`.
+- **Multiple Groupings**: Adding multiple *Group By* selections can split results into more
+  relevant and manageable chunks.
+
+  - Adding :guilabel:`Salesperson` or :guilabel:`Sales Team` breaks up the total count of leads in
+    each :guilabel:`Stage` by who they were assigned to.
+  - Adding :guilabel:`Medium` or :guilabel:`Source` can reveal what marketing avenues generate more
+    sales.
+- **Add Custom Group > Active**: Clicking :menuselection:`Add Custom Group --> Active` separates the
+  results in each stage into *Won (true)* or *Lost (false)*. This can show at what stage leads are
+  won and lost.
```

Comment:
I'd suggest expanding this portion to touch on every single option available. Our doc readers prefer over-explanation. You can do this in a simple list format.

Again, I think this portion should appear above, where you introduce the entire page to the reader...

But I'd probably go with something like...

Filters
-------

The :guilabel:`Filters` section allows users to add custom filters to the results on the
:guilabel:`Pipeline Analysis` page.

- :guilabel:`My Pipeline`: only shows personal leads/opportunities.
- :guilabel:`Opportunities`: only shows opportunities.
- :guilabel:`Leads`: only shows leads.

...and so on...

and then you can start another section for 'Group By' and then 'Favorites'

doing so makes things super-organized and easy to reference